### PR TITLE
Changes active navbar item to primary (green)

### LINKED ISSUE #1149: Fixing the active navbar item
**Describe the bug**
The active navbar-item state looked like a section break in mobile devices

**To Reproduce**
Steps to reproduce the behaviour:
1. Open Navbar Menu in Mobile Devices

**Solution**
Rather than making the active state underlined, we can make the text green without any underline.

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by Nitish145**
Fixes #1149 

#### Describe the changes you have made in this PR -
Changed active navbar item color to primary

### Screenshots of the changes (If any) -
Screenshot from 2020-03-12 17-26-38

### CODE REVIEW

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -45,7 +45,9 @@
     }
 
     &.active {
-      border-bottom: 2px solid $green;
+      a {
```
[the remark this answers is not in the document]
Comment (Nitish145) [reply app/assets/stylesheets/layout.scss]: @nitin10s quadruple nesting was already being used.. Is it ok to by the **navbar-light** level ??

**Comment by Nitish145** [comment]
> This doesn't look much good, can you please make it a little bolder.

Does this work @nitin10s ?
Screenshot from 2020-03-12 22-07-45

**Comment by Ananthavijay** [comment]
Try using the ````green```` instead of ````primary-green````

**Comment by Nitish145** [comment]
> Try using the `green` instead of `primary-green`

I was using `primary` which is same as `green`. What do u suggest @Ananthavijay ?

**Comment by Ananthavijay** [comment]
Try dark green

**Comment by Nitish145** [comment]
Screenshot from 2020-03-12 22-47-38

**Comment by Nitish145** [comment]
@nitin10s any suggestions here ??

**Comment by Nitish145** [comment]
Does this mean to modify *Simulator* border and *on-hover* for each of the items @nitin10s ?

**Comment by nitin10s** [comment]
Nothing to do with Simulator border, and nothing with on-click, just change the colour of active menu item from green to dark green

File: app/assets/stylesheets/layout.scss [hunk 2]
```diff
@@ -37,15 +37,16 @@
...
+  .nav-link {
+    color: $primary;
+    font-weight: 500;
+  }
 
-    &.active {
-      border-bottom: 2px solid $green;
+  &.active {
+    a {
+      color: $button-border-color !important;
```
[the remark this answers is not in the document]
Comment (Nitish145) [reply app/assets/stylesheets/layout.scss]: not using !important requires more specificity than in *bootstrap.cs* which then raises nesting more than 3 error.. What should be done here @nitin10s ?

**Comment by Nitish145** [comment]
@tachyons @nitin10s any idea why after pulling (to resolve merge conflict) tests are failing.. I didn't changed much..

**Comment by Nitish145** [comment]
@nitin10s please check this !!

File: app/assets/stylesheets/layout.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (aayush-05) [reply app/assets/stylesheets/layout.scss:66]: I didn't get you @Nitish145 , won't using pseudo selectors fix this ?

[the remark this answers is not in the document]
Comment (Nitish145) [reply app/assets/stylesheets/layout.scss:66]: @aayush-05 The Problem is that `bootstrap.min.css` uses more specificity to define `.active` color. To overwrite that either we must define even more specificity (which causes nesting more than 3, not allowed by codeclimate) or use `!important` (not allowed by codeclimate either).

`bootstrap.min.css` uses `.navbar-default .navbar-nav > li > a`.

Also, According to **MDN** docs : 
> Only use !important on page-specific CSS that overrides foreign CSS (from external libraries, like Bootstrap or normalize.css).

**Pull request closed without merging**